### PR TITLE
Dolabras

### PR DESCRIPTION
**Pull request by Fitzolir**
#### Summary
Content "Ancient Roman Multitool & Supporting Textbook"

#### Purpose of change
I initially started working on Dolabras to compliment the already existing shovel and entrenching tool digging implements. The `digging` quality felt pretty empty, so I researched and implemented the Dolabra to fluff up the game a little. This helps fill out Museums, Exodii's wares (Cataphracts & Dolabras are the same time period and region), and gives another option for players to put metalworking to good use. Ideally, the Dolabra ends up a handy, well balanced tool for players. 

#### Describe the solution
To compliment the digging tool lineup and pad out other areas, I'm introducing a multitool capable of chopping trees, digging pits, and smashing rock. It's called the Dolabra, and it's as close to a "faithful" reproduction as I could of the real life Dolabra from Ancient Rome. 

I'm also doing this PR with an associated recipe book _Anthropological Metallurgy,_ which is a textbook associated with "ancient" metallurgy recipes that aren't armors or weapons. This way, the value of books like Arms & Armor of Imperial China are spread out more efficiently while opening the door to new neolithic, ancient, and low-tech tools and recipes.

#### Describe alternatives you've considered
During tweaks, the Dolabra originally had cutting 1. This quality would solely so you could write into things, simply as a fluff feature. I've carefully considered all the values of the Dolabra, and I have reasoning behind the choices made about this item. If anyone thinks the Dolabra needs balance changes, feel free to contact me on git or discord with your criticism :) I'd be happy to hear you out and then change the Dolabra accordingly. 

#### Testing
I ran my git repo with .exe and spawned in the Dolabra, and then ensured all of the functions were in order. The same rudimentary testing was performed with the recipe & book. As far as I'm aware, there wasn't a risk of regression involved, but I didn't notice any in my testing regardless. I would attempt putting the Dolabra into different containers to see if there's anything finicky with it about Axe Rings & Leather Belts specifically.

#### Additional context
[DolabraRecipe.txt](https://github.com/CleverRaven/Cataclysm-DDA/files/11644882/DolabraRecipe.txt)
Please find attached a .txt containing my thought process for the somewhat complex recipe. As far as sources you can find them [here](https://oxfordre.com/classics/display/10.1093/acrefore/9780199381135.001.0001/acrefore-9780199381135-e-7260;jsessionid=886FEE1CFB64DA8899626DE65FD1A45C), [here](https://en.wikipedia.org/wiki/Crucible_steel), and [here](https://en.wikipedia.org/wiki/Roman_metallurgy).

### CODE REVIEW

**Comment by Zireael07** [comment]
What is this and why would a survivor in NE know of an obscure Roman era tool??

**Comment by randomtyper** [comment]
> What is this and why would a survivor in NE know of an obscure Roman era tool??

Axepick goes BRRR. Also, book makes monke smart.

**Comment by randomtyper** [comment]
Honestly, I think it would not be unreasonable to make it an autolearn recipe on fabrication 8 or so. Like, if you know what a pick is, and you know what an axe is, combining both in a single tool does not really sound like some nuclear science.

**Comment by Zireael07** [comment]
Sooo.... maybe name it axepick instead? No clue what a dolabras is.... and yes, an axepick sounds like a perfect autofab thing

**Comment by randomtyper** [comment]
> Sooo.... maybe name it axepick instead? No clue what a dolabras is.... and yes, an axepick sounds like a perfect autofab thing

Name it axepick and mention in description that such tool was known in Ancient Rome as dolabra. Add it as learnable early through that historical smithing book (with autolearn as alternative way to get it once you have sufficient skill), as axepicks are pretty obscure in our time.

**Comment by Fitzolir** [comment]
> What is this and why would a survivor in NE know of an obscure Roman era tool??

I wrote what it is in my PR :') It's a pickaxe/axe tool from Ancient Rome responsible for the aqueducts, roads, and most other engineering projects in the nation
Also, it's not an autolearnt recipe. I'll be adding loot tables for the book responsible, but it's a very specific book about a very specific topic that has the recipe.

**Comment by Fitzolir** [comment]
> > Sooo.... maybe name it axepick instead? No clue what a dolabras is.... and yes, an axepick sounds like a perfect autofab thing
> 
> Name it axepick and mention in description that such tool was known in Ancient Rome as dolabra. Add it as learnable early through that historical smithing book (with autolearn as alternative way to get it once you have sufficient skill), as axepicks are pretty obscure in our time.

That exists in real life. That's called a mattock, and it wouldn't be much of a stretch to add a mattock. Also mattocks aren't _obscure_ in our time, you can buy them from most hardware stores including Amazon, Lowes, Home Depot, and the likes.

I set the fabrication at 8 because other tools like shovels, axes, and most notably the pickaxe all require a similar level of skill at approximately Fabrication 8. It's not intended to be an easy recipe for fab 4 as a stopgap for real tools, It's intended to be a higher skill combination of these items. The Dolabra fits in belt loops and the likes as well, and I have a hunch that a pickaxe/shovel/axe/melee smaller than any of those tools on their own would be nerfed into obscurity really fast.

**Comment by Fitzolir** [comment]
> Honestly, I think it would not be unreasonable to make it an autolearn recipe on fabrication 8 or so. Like, if you know what a pick is, and you know what an axe is, combining both in a single tool does not really sound like some nuclear science.

I think this is reasonable. I intend to get this PR merged and then add loot tables for the dolabra and AM (the book). I'm planning on the Exodii to sell it, as well as it spawning naturally in Museums. I'll also be placing the book in various book loot tables & the recipe itself will be distributed throughout other books. I think autolearn on fab 8 might be reasonable but the specific steps involved are a little more involved than "grug smash rock on this side until it chops tree real good" so a reference of some kind is pretty necessary

**Comment by randomtyper** [comment]
> > > Sooo.... maybe name it axepick instead? No clue what a dolabras is.... and yes, an axepick sounds like a perfect autofab thing
> > 
> > 
> > Name it axepick and mention in description that such tool was known in Ancient Rome as dolabra. Add it as learnable early through that historical smithing book (with autolearn as alternative way to get it once you have sufficient skill), as axepicks are pretty obscure in our time.
> 
> That exists in real life. That's called a mattock, and it wouldn't be much of a stretch to add a mattock. Also mattocks aren't _obscure_ in our time, you can buy them from most hardware stores including Amazon, Lowes, Home Depot, and the likes.

This is a mattock:
https://www.amazon.com/Truper-31615-5-Pound-Mattock-36-Inch/dp/B000KL2VFG

It has a pick on one side and adze/hoe on the other.

And here is an axepick:

https://forum.ww2.ru/index.php?showtopic=4637569

It has a pick on one side and an axe head on other. Is it the same as the mattock? And the fact I only found a good example of one after doing a search in Russian means it is somewhat obscure.

And here is Pulaski (or cutter mattock if not made as a firefighting tool):

https://en.m.wikipedia.org/wiki/Pulaski_(tool)

Axe head on one side and adze/hoe on the other.

Which one is dolabra closer to, in your opinion? Just a small question. Mind you, there were at least two styles of dolabras.

> I set the fabrication at 8 because other tools like shovels, axes, and most notably the pickaxe all require a similar level of skill at approximately Fabrication 8. It's not intended to be an easy recipe for fab 4 as a stopgap for real tools, It's intended to be a higher skill combination of these items. The Dolabra fits in belt loops and the likes as well, and I have a hunch that a pickaxe/shovel/axe/melee smaller than any of those tools on their own would be nerfed into obscurity really fast.

You don't have to explain the reasons why you chose exactly that autolearn level to the one who suggested the same level for almost same reasons (axes and picks autolearn by fab 7, fab 8 sounds like decent level to get a combo/summary of the tools known). Fab 8 is pretty close to the theoretical limit of what survivor can make or invent (with skill lv 10 being that limit), so no "grug smashes hunk of metal" is involved. What involved is what I said: "I know what an axe is, and I know what a pick is, and how to make both of those, but it would be convenient to have both tools in one, so I got an idea...". Sure, Intelligence-gated autolearns would be the way to go, but they are not implemented in game.

**Comment by randomtyper** [comment]
> I'm planning on the Exodii to sell it, as well as it spawning naturally in Museums.

Exodii are fine. But why museums? Were there any Ancient Roman settlements on West Coast of USA I do not know of? If there weren't, then such  exhibits are highly unlikely. Even less likely than Vinland viking settlement findings of any sort. And if those are intended to be reproductions, that kinda defeats the purpose of the museums, as they are supposed to be showing historical items of local area (plate armors and other medieval stuff were a major hit in 1800s as decor pieces) rather than make and showcase reproductions of foreign items (and in case of British Museum and other colonialist museums, their exhibits if are reproductions, they are reproductions of the items they have in store and can't exhibit out of concerns like exhibit's integrity).

Hell, just name any West Coast museum that has a dolabra as an exhibit. Metropolitan museum of NY does not count, that was a private collection piece.

**Comment by Fitzolir** [comment]
> Hell, just name any East Coast museum that has a dolabra as an exhibit. Metropolitan museum of NY does not count, that was a private collection piece.
I actually agree with you. I was incorrect to assume there are any museums on the east coast with ancient roman exhibits. Thanks for pointing that out :)

**Comment by Fitzolir** [comment]
> Which one is dolabra closer to, in your opinion? Just a small question. Mind you, there were at least two styles of dolabras.

I would have to say closest to a Pulaski. The Russian Tool does not have a horizontal cutting edge, and the mattock flares too much. Thank you for the question by the way, as it helped me find a great blog from a German Woman with a Masters in Literature and History whom has good information on the topic.

> You don't have to explain the reasons why you chose exactly that autolearn level to the one who suggested the same level for almost same reasons
My mistake, you didn't say why you had come to the same conclusion as me so I thought it would be best for me to state my reasoning. We can both agree on autolearn level 8, so I think it'd just be best if I retconned the book and simply made it an autolearn recipe.

**Comment by randomtyper** [comment]
> > Hell, just name any East Coast museum that has a dolabra as an exhibit. Metropolitan museum of NY does not count, that was a private collection piece.
>  I actually agree with you. I was incorrect to assume there are any museums on the east coast with ancient roman exhibits. Thanks for pointing that out :)
> 
> EDIT: There is a museum in Northeim Germany that has dolabra, specifically from the ancient fort Hedemunden. Obviously, this is nowhere near New England, but museums with dolabra do exist.

Indeed, it makes sense to have a dolabra in museum on the lands that once had extensive contact with Rome. If game was set in almost any country in Europe, I wouldn't bat an eyelid.



> > Which one is dolabra closer to, in your opinion? Just a small question. Mind you, there were at least two styles of dolabras.
> 
> I would have to say closest to a Pulaski. The Russian Tool does not have a horizontal cutting edge, and the mattock flares too much. Thank you for the question by the way, as it helped me find a great blog from a German Woman with a Masters in Literature and History whom has good information on the topic.

The thing is that there are at least two versions of dolabrae, with differing non-axe heads (and I saw tools with only hoe heads (without axe) classified as dolabrae for some reason), imperial and republican (and I can't really figure out which is which):

https://ru.pinterest.com/pin/432275264229489934/

One design has an adze-like head, other has a pick-like one.

Hell, even Hedemunden dolabrae were of different designs, if you noticed: one design with adze-like head, and one with rear more befitting of a very long and narrow axe or a verticaly extended pick:

https://upload.wikimedia.org/wikipedia/commons/8/80/HedemuePionieraexte1.jpg

Seems like pick-like ones were of imperial design, though, or at least used in Britain in imperial times (pick ends may be flattened due to work damage):

https://www.nms.ac.uk/explore-our-collections/stories/scottish-history-and-archaeology/the-romans-in-scotland/sections-1/romans-in-scotland-trimontium-museum-loan/
https://www.nms.ac.uk/explore-our-collections/collection-search-results/pickaxe-pioneers/135790

Her is another set of (mostly) pick-like dolabrae:
https://64.media.tumblr.com/682285577094f2ba2b20b3b7260e37d8/tumblr_p7g1j5SQx51rwjpnyo2_540.jpg

Most reproductions use the pick-like dolabra as basis, too:

https://www.celticwebmerchant.com/en/roman-dolabra.html

The practical difference between designs is that a pick-like head is much better at dealing with solid rock due to smaller striking surface (giving valid explanation for dig through terrain ability) while adze is mostly for dig 1 (basically a small hoe) and crude woodwork (chop 1) if sharpened (something that already present axe head can do).

Also, plural for dolabra is dolabrae. Good thing that roman centurions are so rare now.

**Comment by randomtyper** [comment]
> > Which one is dolabra closer to, in your opinion? Just a small question. Mind you, there were at least two styles of dolabras.
> 
> I would have to say closest to a Pulaski. The Russian Tool does not have a horizontal cutting edge, and the mattock flares too much. Thank you for the question by the way, as it helped me find a great blog from a German Woman with a Masters in Literature and History whom has good information on the topic.
> 
> > You don't have to explain the reasons why you chose exactly that autolearn level to the one who suggested the same level for almost same reasons

That said, can you link me that blog? Sounds like an interesting read.

**Comment by randomtyper** [comment]
> I would have to say closest to a Pulaski. The Russian Tool does not have a horizontal cutting edge, and the mattock flares too much. Thank you for the question by the way, as it helped me find a great blog from a German Woman with a Masters in Literature and History whom has good information on the topic.

And the "Russian tool" is not Russian, but an Austro-Hungarian WW1 era piece of equipment:

https://www.iwm.org.uk/collections/item/object/30113224

It would fill basically the same role in A-H army as dolabra in Ancient Roman, I guess.

File: data/json/recipes/tools/tools_hand.json [hunk 1]
```diff
@@ -1603,5 +1603,50 @@
...
+  "qualities": [
+    { "id": "HAMMER", "level": 3 },
+    { "id": "ANVIL", "level": 2 },
+    { "id": "SAW_W", "level": 2 },
+    { "id": "GRIND", "level": 2 },
+    { "id": "FILE", "level": 1 }
+  ],
+  "using": [
+    [ "forging_standard", 10 ],
+    [ " steel_standard", 10 ]
```
Comment (Procyonae) [inline data/json/recipes/tools/tools_hand.json:1638]:
```suggestion
    [ "steel_standard", 10 ]
```

**Comment by Procyonae** [comment]
It feels like you could do with at least two versions of this, the ancient one sold by the exodii and then separately a craftable steel version with a more generic description, I also don't understand the museum spawn (just bc some current things that spawn in the museum don't make sense for New England doesn't mean more should be added, although the craftable version as just a replica ancient tool might be reasonable) nor why the recipe would appear in the eastern books if it's meant to specifically be the ancient roman tool.
Other than that I'd just add the spawns to this PR and add some existing recipes to your new book if there are any appropriate ones bc it doesn't make much sense for them to be added unobtainable.

**Comment by Fitzolir** [comment]
> It feels like you could do with at least two versions of this, the ancient one sold by the exodii and then separately a craftable steel version with a more generic description, I also don't understand the museum spawn (just bc some current things that spawn in the museum don't make sense for New England doesn't mean more should be added, although the craftable version as just a replica ancient tool might be reasonable) nor why the recipe would appear in the eastern books if it's meant to specifically be the ancient roman tool. Other than that I'd just add the spawns to this PR and add some existing recipes to your new book if there are any appropriate ones bc it doesn't make much sense for them to be added unobtainable.

Thanks for bringing this up! I'm going to close the PR for now so I can put it back in the oven and I'll come back with it more fleshed out and correct the most glaring issues. ((Syntax))

**Pull request closed without merging**